FLOWTYPE.md

```diff
@@ -1,21 +1,27 @@
-Using flowtype
-==============
+# Using flowtype
 
 This codebase has been set up to use [flowtype](http://flowtype.org/), a static typechecker for Javascript.
 
-## Setup
+## Installation
 
-To use it, first install `flow`:
+To use it, first install `flow`.
+
+### *nix
 
 ``` sh
 brew update
 brew install flow
```

Comment:
Shouldn't we keep `flow-bin` for *nix systems? (I liked the idea since you've now added a section for Windows users to download and install windows binaries). I noticed flow-bin in the package.json if I'm not mistaken?